src/IJulia/ijulia.js

```diff
@@ -24,16 +24,19 @@
             var _src = 'data:' + type + ';base64,' + val;
 	    $(container).find("img").attr('src', _src);
 	    break;
+	case "text/latex":
+		if (MathJax){
+			var math = MathJax.Hub.getAllJax(container)[0];
```

Comment:
Gets the MathJax elements inside the output block: https://docs.mathjax.org/en/v2.5-latest/typeset.html#manipulating-individual-math-elements

Slightly paraphrasing: This looks up the list of math elements in the [container] element (I'm assuming there is only one) and takes the first one (element 0) and stores it in math. This is an element jax object (see the Element Jax specification for details), which has a Text() method that can be used to set the input text of the math element, and retypeset it.